dvc/repo/get.py

```diff
@@ -17,6 +20,19 @@
 logger = logging.getLogger(__name__)
 
 
+def _copy_git_file(repo, src, dst):
+    src_full_path = os.path.join(repo.root_dir, src)
+    dst_full_path = os.path.abspath(dst)
+
+    if repo.root_dir not in src_full_path:
```

Comment:
repo.root_dir is `/path/to/dir` and not `/path/to/dir/`, so it will break on `src_full_path` looking like `/path/to/dir-some-name`. Please use path_isin from dvc/utils. 🙂